### DISCUSSION TITLE
Replacements for legacy components

### DISCUSSION THREAD

**Discussion by hakimio** [taiga-family/taiga-ui#8721] [Q&A] [answered]
Do you have some documentation listing intended replacements for "legacy" components?

**Comment by waterplea** [comment, marked as answer]
So far only `InputPhoneInternational` was rewritten with next textfields, but eventually they all will be.

**Reply by hakimio** [reply]
So, basically the components are deprecated but almost all don't have any replacements yet?

**Reply by waterplea** [reply]
They are not deprecated, once you see them actually deprecated with a strikethrough like in `InputPhoneInternational` — that means the replacement is ready. We will also try to provide migrations with them.
